### PR TITLE
 login page UI issue were resolved

### PR DESCRIPTION
**Pull request by HimanshuNarware**
## Description 📝
Align all the element  and added animation and hover effect on the button also added colors to the forgot password button , create new account, The login button UI issue were resolved.

Fixes/Closes: <#209>

## Changes 💬

- Added animation effects on the login page
- Added hover effects on the login page
- Added color properties to the button of the login page
- Align the heading of the login page

## Additional Notes 📒

Include any additional information or context that might be helpful for the reviewers. 
***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***

### CODE REVIEW

File: pages/login.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { DarkModeToggle } from "../Components/components";
 import { AuthContext } from "@/context/AuthContext";
 import { ToastContainer, toast } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
+import "./signup";
```
Comment (lalitmee) [inline pages/login.tsx:10]: Why do we need this?

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

File: pages/login.tsx [hunk 2]
```diff
@@ -166,12 +167,13 @@ const Login = () => {
 
               {/* Create Account */}
               <div className="flex flex-col items-center justify-between mt-4 text-sm">
-                <p className="w-fit text-slate-600 dark:text-gray-100">
+                <p className="w-fit text-slate-900 dark:text-sky-600">
```
[the remark this answers is not in the document]
Comment (HimanshuNarware) [reply pages/login.tsx:170]: ok

File: pages/login.tsx (hunk 1 above)
Comment (HimanshuNarware) [reply pages/login.tsx]: @lalitmee by mistake, I had added this file but I removed it now

File: pages/login.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (HimanshuNarware) [reply pages/login.tsx]: @PiyushKalyanpy revert back all colour changes successfully

**Comment by PiyushKalyanpy** [comment]
Hey @HimanshuNarware as we have updated our login and signup designs, we cannot accept these changes due to conflicts. If you can add dark mode in new auth pages, then please create an issue.

**Pull request closed without merging**